src/server/watchGuard/watchGuard.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference types="node" />
+
+if (process.argv.length < 3) {
+    process.exit(1);
+}
+const directoryName = process.argv[2];
+const fs: { watch(directoryName: string, options: any, callback: () => {}): any } = require("fs");
+fs.watch(directoryName, { recursive: true }, () => ({})).close();

```

Comment:
I could think of a number of scenarios where this might be more defensive (e.g. check it exists, check it is a dir, check fs.watch returned a watcher object, etc...), but in all those cases seems the resulting error would result in correct behavior anyway in the listening process. However, might be worth a comment to note that this is deliberately "loose" here.